docs/schemas/events/metadatamanager/example/MSC__example1.json

```diff
@@ -0,0 +1,28 @@
+{
+  "version": "0",
+  "id": "f71bbbbb-5b36-40c2-f7dc-804ca6270cd6",
+  "detail-type": "MetadataStateChange",
+  "source": "orcabus.metadatamanager",
+  "account": "123456789012",
+  "time": "2024-05-01T09:25:44Z",
+  "region": "ap-southeast-2",
+  "resources": [],
+  "detail": {
+    "model": "LIBRARY",
+    "action": "CREATE",
+    "refId": "lib.01FZ7N3XK1GZ5V9P8Z4H7J5S9F",
+    "data": {
+      "orcabus_id": "lib.01FZ7N3XK1GZ5V9P8Z4H7J5S9F",
```

Comment:
Would you be able to support `camelCase` for event alignment? Convention is the same as API, once broadcast out from the app...

Reply:
Yes, will look into this. thanks!